myproject/main.php

```diff
@@ -19,27 +19,18 @@
 <form>
 <input type="text" name="parametr" >
 <input type="submit">
-<table>
-    <?php for ($i=0; $i<count($list); $i++):?>
-    
+<table border="1">
+    <?php for ($i=0; $i<count($list); $i++)
```

Comment:
Does it work correctly now?
Tag `<tr>` should be in the first cycle, but not in the second.